docs/status.md

```diff
@@ -112,7 +112,7 @@ one of the following tracking labels.
 | replica_id               |      no       |     yes*     | yes(need-revisit) |        yes        |     no      |
 | reshape                  |      yes      |     yes      |    infeasible     |        yes        |     yes     |
 | return                   |      no       |     yes*     |        no         |        yes        |     no      |
-| reverse                  |      no       |     yes*     |       yes*        |        no         |     no      |
+| reverse                  |      yes      |     yes*     |       yes*        |        no         |     no      |
```

Comment:
Have you checked the verification and type inference columns? If they are not in sync, let's create a ticket following the [protocol](https://github.com/openxla/stablehlo/blob/main/docs/spec_checklist.md).

Reply:
The verification and type inference are in sync with the spec draft, I should make them yes according to the protocol but I did not see any other values in these columns with a yes, which makes me wonder if I am missing anything.

Reply:
#273 
Created an issue to add verification for ReverseOp

Reply:
Nice catch re: the missing verifier!